docs/user/general/identifiers.rst

```diff
@@ -176,3 +176,89 @@ Query delimited multiple indices seperated by ``,``::
     |-------|
     | 5     |
     +-------+
+
+
+
+Fully Qualified Table Names
+===========================
+
+Description
+-----------
+With the introduction of different datasource catalogs along with Opensearch, support for fully qualified table names became compulsory to resolve tables to a catalog.
+
+Format for fully qualified table name.
+``<catalogName>.<schemaName>.<tableName>``
+
+* catalogName:[Mandatory] Catalog information is mandatory when querying over tables from catalogs other than opensearch connector.
+
+* schemaName:[Optional] Schema is a logical abstraction for a group of tables. In the current state, we only support ``default`` and ``information_schema``. Any schema mentioned in the fully qualified name other than these two will be resolved to be part of tableName.
+
+* tableName:[Mandatory] tableName is mandatory.
+
+The current resolution algorithm works in such a way, the old queries on opensearch work without specifying any catalog name.
+So queries on opensearch indices doesn't need a fully qualified table name.
+
+Table Name Resolution Algorithm.
+--------------------------------
+
+Fully qualified Name is divided into parts based on ``.`` character.
+
+TableName resolution algorithm works in the following manner.
+
+1. Take the first part of the qualified name and resolve it to a catalog from the list of catalogs configured.
+If it doesn't resolve to any of the catalog names configured, catalog name will default to ``@opensearch`` catalog.
+
+2. Take the first part of the remaining qualified name after capturing the catalog name.
+If this part represents any of the supported schemas under catalog, it will resolve to the same otherwise schema name will resolve to ``default`` schema.
+Currently ``default`` and ``information_schema`` are the only schemas supported.
+
+3. Rest of the parts are combined to resolve tablename.
+
+** Only table name identifiers are supported with fully qualified names, identifiers used for columns and other attributes doesn't require prefixing with catalog and schema information.**
+
+Examples
+--------
+Assume [my_prometheus] is the only catalog configured other than default opensearch engine.
+
+1. ``my_prometheus.default.http_requests_total``
+
+catalogName = ``my_prometheus`` [Is in the list of catalogs configured].
+
+schemaName = ``default`` [Is in the list of schemas supported].
+
+tableName = ``http_requests_total``.
+
+2. ``logs.12.13.1``
+
+
+catalogName = ``@opensearch`` [Resolves to default @opensearch connector, as prometheus is not in the list of catalogs configured].
```

Comment:
IMO the description text needs to be - "Resolves to default `@opensearch` connector since [my_prometheus] is the only catalog configured name .

Reply:
+1